### DISCUSSION TITLE
Loading spinner randomly stays active on custom page

### DISCUSSION THREAD

**Discussion by SeriousKen** [filamentphp/filament#7722] [Help] [answered]
I'm just playing around with custom pages in V3 and just created a simple counter page and noticed odd behaviour where the loading spinner on `x-filament::button` seems to stay active sometimes.

My page object is

```php
namespace App\Filament\Pages;

use Filament\Pages\Page;

class Counter extends Page
{
    public $counter = 0;

    protected static ?string $navigationIcon = 'heroicon-o-document-text';

    protected static string $view = 'filament.pages.counter';

    public function increment()
    {
        $this->counter++;
    }

    public function decrement()
    {
        $this->counter--;
    }
}
```

And the view is:

```blade
<x-filament-panels::page>
    <input wire:model="counter" />

    <x-filament::button wire:click="increment" icon="heroicon-m-plus"></x-filament::button>
    <x-filament::button wire:click="decrement" icon="heroicon-m-minus"></x-filament::button>

    <div wire:loading>Updating counter...</div>
</x-filament-panels::page>
```

Am I missing something or is this a bug?

**Comment by danharrin** [comment, marked as answer]
Yeah it's a Livewire bug I believe, I think its open as an issue here

**Reply by ousid** [reply]
Is there a workaround solution? This issue still persists even in the FI `3.x`

**Reply by danharrin** [reply]
I haven't experienced it in a few months since updating Livewire

**Reply by ousid** [reply]
Yeah, this issue still exists, and you notice when you added `@livewireStyles` above `@filamentStyles` the spinners disappear, but won't function as expected
